sdk/python/kfp/registry/registry_client_test.py

```diff
@@ -48,10 +48,10 @@ def test_is_ar_host_kfp(self):
         self.assertEqual(client._is_ar_host(), True)
 
     def test_is_ar_host_error(self):
```

Comment:
nit: looks like this test name is out of date (no longer tests error)

Reply:
Good catch. I updated it.